pkg/pool-manager/pool.go

```diff
@@ -212,3 +213,16 @@ func (p *PoolManager) isInstanceOptedIn(namespaceName, mutatingWebhookConfigName
 
 	return false, nil
 }
+
+func GetMacPoolSize(rangeStart, rangeEnd net.HardwareAddr) (uint64, error) {
+	err := checkRange(rangeStart, rangeEnd)
+	if err != nil {
+		return 0, errors.Wrap(err, "mac Pool Size  is negative")
+	}
+
+	//making sure Uint64 receives an byte slice of size 8
```

Comment:
Let's add a helper innter function with
```golang
addTwoBytesPadding = func(....
startInt := addTwoBytesPadding(rangeStart
endInt := addTwoBytesPadding(rangeEnd
```

Reply:
if it's ok I'll use the function Or suggested: 1972f47#diff-baa41b07e1d50dce50eab577f6b940d5R125